Utilities/StaticAnalyzers/scripts/callgraph.py

```diff
@@ -57,17 +57,17 @@
 m2=vre.search(function2)
 assert(m2)
 
-modules = list()
-with open('modules_in_ib.txt', 'r') as m:
-    for line in m:
-        modules.append(line.strip())
-mods='|'.join(modules)
-mods_regex=r'\b(' + mods+ r'QQQQQQQQ)\b'
-comp_mods_regex=re.compile(mods_regex)
-n=comp_mods_regex.search(function)
-assert(n)
-n2=comp_mods_regex.search(function2)
-assert(n2)
+#modules = list()
+#with open('modules_in_ib.txt', 'r') as m:
+#    for line in m:
+#        modules.append(line.strip())
+#mods='|'.join(modules)
+#mods_regex=r'\b(' + mods+ r'QQQQQQQQ)\b'
+#comp_mods_regex=re.compile(mods_regex)
+#n=comp_mods_regex.search(function)
+#assert(n)
+#n2=comp_mods_regex.search(function2)
+#assert(n2)
```

Comment:
Is it useful to keep these in comments (wrt. digging out from git history)?

Reply:
No